resources/lang/vendor/prequel/de/lang.php

```diff
@@ -0,0 +1,164 @@
+<?php
```

Comment:
Since we are going with the convention of generating vendor files & not storing them, these should be removed from VCS as well.

Reply:
Leaving this as a general note since we decided to remove prequel:
vendor subfolders in the `resources` folder are not for generated files, they are considered as override files for your composer packages. As such it would be bad practice to gitignore them. Their intended use is for modifying css, blade, vue, etc. files that are package developers define as default template or even some laravel components (like the mail & notification assets)

Typically you would only publish these when you explicitly intend to override those files, however the developer of the prequel package has decided that to publish these files during package installation.